Documentation/deploy-master.md

```diff
@@ -154,6 +138,7 @@ spec:
     - --tls-private-key-file=/etc/kubernetes/ssl/apiserver-key.pem
     - --client-ca-file=/etc/kubernetes/ssl/ca.pem
     - --service-account-key-file=/etc/kubernetes/ssl/apiserver-key.pem
+    - --runtime-config=extensions/v1beta1/deployments=true,extensions/v1beta1/daemonsets=true,extensions/v1beta1=true,extensions/v1beta1/thirdpartyresources=true
```

Comment:
I think that `v1beta/deployments` and `v1beta/daemonsets` are enabled by default in v1.2

https://github.com/kubernetes/kubernetes/blob/release-1.2/pkg/master/master.go#L667